### PR TITLE
add files for jetstream login

### PR DESCRIPTION
**Pull request by sateeshperi**
updated jetstream login screenshots

PR author:
- [ ] pull request is ready for review & merge

Things to check:

- [ ] tutorial links to previous tutorial (at top) and next tutorial (at bottom)
- [ ] tutorial has learning objectives at top

### CODE REVIEW

File: jetstream.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
+Accessing The Jetstream Cloud
+===
```
Comment (taylorreiter) [inline jetstream.md:2]: @sateeshperi please add objectives

File: jetstream.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+What we're going to do here is walk through starting up a running
+computer (an "instance") on the Jetstream service. 
+
+[Jetstream](https://jetstream-cloud.org/) is an [NSF/XSEDE] resource designed to promote and provide configurable cyberinfrastructure in the form of cloud computing to both novice and experienced users.
+
+If you would like to read more about cloud computing, see this [Carpentry Cloud Computing lesson](http://www.datacarpentry.org/cloud-genomics/01-why-cloud-computing/).
+
+- This is the first and last place in these lessons where it will matter if you are using PC, Mac, or Linux. After we connect to our virtual machines built using the same image; we will all be on the same operating system/computing environment.
```
Comment (taylorreiter) [inline jetstream.md]: This is not accurate. It matters for `scp` in the QA/QC lesson

Comment (taylorreiter) [reply jetstream.md]: Also it mattered for copy-paste functionality

File: jetstream.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+
+> **NOTE: WINDOWS users** will need to install a UNIX-ready terminal to SSH-login (if you haven't already). We recommend Git bash. See workshop setup page for instructions.
```
Comment (taylorreiter) [inline jetstream.md]: We should note that >windows 10 comes with an ubuntu terminal.

File: jetstream.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+
+	+	Provider: "Jetstream - TACC"
+
+	+ Instance size: We recommend ""m1.medium" (CPU: 6, Mem: 16GB, Disk: 60GB)"for this tutorial; Though  depending on your allocations, choose most suitable one.**
+
+![](jetstream/images/login8.png)
+
+- **Launch instance and wait for the build to be deployed (~ 5-10 minutes).**
+
+> Note: During the build process: `scheduling-->building-->spawning-->deploying-->Networking-->N/A`; Be patient! Don't reload!. Once the virtual machine is ready, the "Activity" column will show "N/A" and the "Status" column will turn green and "Active".
```
Comment (taylorreiter) [inline jetstream.md]: double punctuation after `reload!.`

File: jetstream.md [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+
+- **Navigate back to 'Projects' and click on your new instance's name to see more information related to the instance you just created! and Copy the IP address of your instance created.**
+
+![](jetstream/images/login10.png)
+
+> **Great! We have now built our very own remote virtual machine with all the software pre-installed. Next we will use SSH-Secure-Login to access these remote instances from our laptop !!!.**
+
+# SSH Secure-Login
+
+- MACOS & LINUX users can open a Terminal window now.
```
Comment (taylorreiter) [inline jetstream.md]: macOS

File: jetstream.md [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+- Windows users start a new session in git bash
```
Comment (taylorreiter) [inline jetstream.md]: i think if they have windows 10, using the ubuntu distribution is a lot better than gitbash bc of `ming64` user things

**Review by taylorreiter — CHANGES_REQUESTED** [review]

File: jetstream.md (hunk 4 above)
Comment (AstrobioMike) [reply jetstream.md]: What does don’t reload mean, @sateeshperi? Reloading the page doesn’t affect the build

File: conda_tutorial.md [hunk 7]
```diff
@@ -0,0 +1,184 @@
...
+```
+
+Similarly to the conda listcommand, this one results in a list of the matching package names, versions, and channels:
+
+NEED screenshot
+
+To install a package (for instanceseaborn) that is inside a channel that is on your channel list, run this command (if you don’t specify which version you want, it’ll automatically install the latest available version from the highest priority channel):
+
+```
+conda install checkm-genome
```
Comment (taylorreiter) [inline conda_tutorial.md]: can we use a different example package, or will we be using this? checkm requires py2 I think, and should usually be installed into it's own env. something like samtools could be a good option here.

File: jetstream.md [hunk 8]
```diff
@@ -0,0 +1,136 @@
...
+| Report | instance exhibiting unexpected behavior? Report here |
+| Image | Request an image (a type of template for a virtual machine) of a running instance |
+| Suspend | Suspending an instance frees up resources for other users and allows you to safely preserve the state of your instance without imaging. Your time allocation no longer counts against you in the suspended mode |
+| Shelve | Shelving will safely preserve the state of your instance for later use. And, it frees up resources for other users . In fact, it is the best way to reduce resource usage when compared with other actions, like "suspend" and "stop".Your time allocation no longer counts against you in the shelved mode |
+| Stop | Stop an instance |
+| Reboot | Reboot an instance |
+| Redeploy | Redeploying an instance will allow you to fix instances that show up as 'active - deploy_error' |
+| Delete | following instance will be shut down and all data will be permanently lost |
+
+> **Note: It is advisable to delete the machine if you are not planning to use it in future to save valuable resources. However if you want to use it in future, you can suspend it. Notice: IP address changes**
```
Comment (taylorreiter) [inline jetstream.md]: stop and suspend each use like 25 or 50% of the resources as a running instance. you need to shelve an instance if you want to not use any SUs.

File: jetstream.md [hunk 9]
```diff
@@ -0,0 +1,136 @@
...
+
+	+	Provider: "Jetstream - TACC"
+
+	+ Instance size: We recommend ""m1.medium" (CPU: 6, Mem: 16GB, Disk: 60GB)"for this tutorial; Though  depending on your allocations, choose most suitable one.**
+
+![](jetstream/images/login8.png)
+
+- **Launch instance and wait for the build to be deployed (~ 5-10 minutes).**
+
+> ***Note: During the build process: `scheduling-->building-->spawning-->deploying-->Networking-->N/A`; Be patient! Don't reload!!. Once the virtual machine is ready, the "Activity" column will show "N/A" and the "Status" column will turn green and "Active".***
```
Comment (taylorreiter) [inline jetstream.md]: I reload all the time during the build process...is there documentation on the jetstream page that says I should stop doing this?

File: jetstream.md [hunk 10]
```diff
@@ -0,0 +1,136 @@
...
+
+
+What we're going to do here is walk through starting up a running
+computer (an "instance") on the Jetstream service. 
+
+[Jetstream](https://jetstream-cloud.org/) is an [NSF/XSEDE] resource designed to promote and provide configurable cyberinfrastructure in the form of cloud computing to both novice and experienced users.
+
+If you would like to read more about cloud computing, see this [Carpentry Cloud Computing lesson](http://www.datacarpentry.org/cloud-genomics/01-why-cloud-computing/).
+
+**NOTE: WINDOWS users** will need to install a UNIX-ready terminal to SSH-login (if you haven't already).
```
Comment (taylorreiter) [inline jetstream.md:17]: beautiful explanation and options :)

File: conda_tutorial.md [hunk 11]
```diff
@@ -0,0 +1,184 @@
...
+# What is Conda?
+
+Conda is open source package and runs on Windows, Mac OS and Linux.
+
++ Conda can quickly install, run, and update packages and their dependencies.
++ Conda can create, save, load, and switch between project specific software environments on your local computer.
+
+![](_static/conda1.png)
+
+Conda as a package manager helps you find and install packages. If you need a package that requires a different version of Python, you do not need to switch to a different environment manager, because Conda is also an environment manager. With just a few commands, you can set up a totally separate environment to run that different version of Python, while continuing to run your usual version of Python in your normal environment.
```
Comment (taylorreiter) [inline conda_tutorial.md:34]: we mention that conda is an environment manager before defining an environment

File: conda_tutorial.md [hunk 12]
```diff
@@ -0,0 +1,184 @@
...
+```
+
+## Activating and leaving (deactivating) an environment
+
+Inside a new Conda installation, the root environment is activated by default, so you can use it without activation.
+
+In other cases, if you want to use an environment (for instance manage packages, or run Python scripts inside it) you need to first activate it.
+
+```
+source activate mynewenv
```
Comment (taylorreiter) [inline conda_tutorial.md]: this had changed. with newer versions of conda this should be `conda activate mynewenv`

File: conda_tutorial.md [hunk 13]
```diff
@@ -0,0 +1,184 @@
...
+
+To create a new environment named, for instance mynewenv (you can name it what ever you like), that includes, let’s say, a Python version 3.4., run:
+
+```
+conda create --name mynewenv python=3.4
+```
+
+## Activating and leaving (deactivating) an environment
+
+Inside a new Conda installation, the root environment is activated by default, so you can use it without activation.
```
Comment (taylorreiter) [inline conda_tutorial.md:61]: can you add an example of `(base)` in the prompt when the default env is activated?

File: conda_tutorial.md [hunk 14]
```diff
@@ -0,0 +1,184 @@
...
+
+```
+source activate mynewenv
+```
+The command prompt changes upon the environment’s activation, it now contains the active environment’s name.
+
+The directories of the active environment’s executable files are added to the system path (this means that you can now access them more easily). You can leave an environment with this command:
+
+```
+source deactivate
```
Comment (taylorreiter) [inline conda_tutorial.md]: `conda deactivate`

File: conda_tutorial.md [hunk 15]
```diff
@@ -0,0 +1,184 @@
...
+```
+
+It needs to be mentioned that upon deactivating an environment, the root environment becomes active automatically.
```
Comment (taylorreiter) [inline conda_tutorial.md]: the base environment becomes active automatically, not the root env

File: conda_tutorial.md [hunk 16]
```diff
@@ -0,0 +1,184 @@
...
+
+
+## What are Conda channels?
+
+Channels are the locations of the repositories where Conda looks for packages. Upon Conda’s installation, Continuum’s (Conda’s developer) channels are set by default, so without any further modification, these are the locations where your Conda will start searching for packages.
```
Comment (taylorreiter) [inline conda_tutorial.md]: we should probably add more explanation about what a repository is (like that it's not on your instance or on your local computer, but that it's on the internet somewhere)

File: conda_tutorial.md [hunk 17]
```diff
@@ -0,0 +1,184 @@
...
+
+Channels exist in a hierarchical order. The channel with the highest priority is the first one that Conda checks, looking for the package you asked for. You can change this order, and also add channels to it (and set their priority as well).
```
Comment (taylorreiter) [inline conda_tutorial.md]: i'm not sure its hierarchical...just ordered? I mean they are sort of hierarchical but nothing can be at the same level, so I think ordered might be more clear.

File: conda_tutorial.md [hunk 18]
```diff
@@ -0,0 +1,184 @@
...
+
+![](_static/conda4.png)
+
+If multiple channels contain a package, and one channel contains a newer version than the other one, the channels’ hierarchical order determines which one of these two versions are going to be installed, even if the higher priority channel contains the older version.
```
Comment (taylorreiter) [inline conda_tutorial.md]: the order of the channels determines which one...

File: conda_tutorial.md [hunk 19]
```diff
@@ -0,0 +1,184 @@
...
+
+![](_static/conda5.png)
+
+
+### Bioconda Channel
```
Comment (taylorreiter) [inline conda_tutorial.md]: need to make a comment that bioconda is not available for windows systems

File: conda_tutorial.md [hunk 20]
```diff
@@ -0,0 +1,184 @@
...
+
+```
+conda search -f checkm-genome
+```
+
+Similarly to the conda listcommand, this one results in a list of the matching package names, versions, and channels:
+
+NEED screenshot
+
+To install a package (for instanceseaborn) that is inside a channel that is on your channel list, run this command (if you don’t specify which version you want, it’ll automatically install the latest available version from the highest priority channel):
```
Comment (taylorreiter) [inline conda_tutorial.md]: what is `instanceseaborn`?

File: conda_tutorial.md [hunk 21]
```diff
@@ -0,0 +1,184 @@
...
+
+```
+conda install checkm-genome
+```
+
+You can also specify the package’s version:
```
Comment (taylorreiter) [inline conda_tutorial.md:147]: we should probably add a comment somewhere about why versions are important, and what they are

**Review by taylorreiter — CHANGES_REQUESTED** [review]
It's starting to look really good, some small changes and definitions requested

File: conda_tutorial.md [hunk 22]
```diff
@@ -0,0 +1,184 @@
...
+
+
+
+## Installing & Activating Conda
+We have already installed conda on this instance, but please see the [installation HackMD](https://hackmd.io/im0eDxViQgmxTL4zYBfGoQ) after the class if you'd like to do this yourself.
+
+To be able to use that installation, we need to let the instance know what path to find bioconda in:
+
+```{bash}
+echo export PATH=$PATH:/opt/miniconda3/bin >> ~/.bashrc
```
Comment (taylorreiter) [inline conda_tutorial.md]: this should happen automatically in the conda installation process...does it not?

File: conda_tutorial.md [hunk 23]
```diff
@@ -0,0 +1,184 @@
...
+Try running a program pre-installed on this instance:
+
+```
+fastqc
+```
+
+To search for all the available versions of a certain package, you can use the search command. For instance, to list out all the versions of the seaborn package (it is a tool for data visualization), run:
+
+```
+conda search -f checkm-genome
```
Comment (taylorreiter) [inline conda_tutorial.md]: can you change this to samtools as well?

**Review by taylorreiter — CHANGES_REQUESTED** [review]
so close!

File: conda_tutorial.md (hunk 13 above)
Comment (sateeshperi) [reply conda_tutorial.md:61]: i mentioned it towards the end after the `conda deactivate` command. did you want how it looks in the shell separately?

Comment (taylorreiter) [reply conda_tutorial.md:61]: that's what i was thinking but it's not a huge deal

Comment (taylorreiter) [reply conda_tutorial.md:61]: not a big deal, it's probably fine

**Review by taylorreiter — APPROVED** [review]
LGTM

**Pull request merged**